content/sensu-go/5.2/reference/license.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "License management"
+linkTitle: "License Management"
+description: "Activate and manage your enterprise license for Sensu Go."
+weight: 100
+version: "5.2"
+product: "Sensu Go"
+menu:
+  sensu-go-5.2:
+    parent: reference
+---
+
+- [Activating your license](#enterprise-features-in-sensu-go)
+- [License expiration](#license-expiration)
+- [License management API](../../api/license)
+
+### Quick links
+
+- [Log in to your Sensu account](https://account.sensu.io/)
+- [Configure LDAP authentication](../../installation/auth)
```

Comment:
I got a 404 on this link.. I think this might resolve it?
```suggestion
- [Configure LDAP authentication](../installation/auth)
```

Reply:
I'm actually not sure if that will resolve it the error, but digging around now!